docs/example.md

```diff
@@ -109,6 +109,7 @@ This includes device specific information like operating system, manufacturer, o
 | `withDataCollectionLevel` | sets the data collection level (enum DataCollectionLevel) | USER_BEHAVIOR |
 | `withCrashReportingLevel` | sets the crash reporting level (enum CrashReportingLevel) | OPT_IN_CRASHES |
 | `enableVerbose`  | enables extended log output for OpenKit if the default logger is used  | `false` |
+| `withLogLevel` | sets the default log level if the default logger is used | `LogLevel.WARN` |

```

Comment:
This is a point for discussion: Regarding the wording "default log level"... probably better would be something indicating that this is a "threshold" to include or exclude a log statement. Or simply skip the "default" in "default log level". This is used in a variety of places.

Reply:
Discussed in real life - and found a solution that fits both.